plugins/netskope/icon_netskope/actions/upload_json_config/action.py

```diff
@@ -0,0 +1,17 @@
+import insightconnect_plugin_runtime
+
+from .schema import Component, Input, Output, UploadJsonConfigInput, UploadJsonConfigOutput
+
+
+class UploadJsonConfig(insightconnect_plugin_runtime.Action):
+    def __init__(self):
+        super(self.__class__, self).__init__(
+            name="upload_json_config",
+            description=Component.DESCRIPTION,
+            input=UploadJsonConfigInput(),
+            output=UploadJsonConfigOutput(),
+        )
+
+    def run(self, params={}):
+        response = self.connection.client.upload_json_config(**params.get(Input.URLLIST))
```

Comment:
What does the `**` represent here for `params.get`?

Reply:
It's just unpacking `urllist` object that contains `content` and `filename` fields in it directly to the function `upload_json_config`.